src/toil/wdl/wdltoil.py

```diff
@@ -1548,7 +1532,7 @@ def patched_run_invocation(*args: Any, **kwargs: Any) -> List[str]:
         # container-determined strings that are absolute paths to WDL File
         # objects, and like MiniWDL we can say we only support
         # working-directory-based relative paths for globs.
-        outputs_library = ToilWDLStdLibTaskOutputs(file_store, host_stdout_txt, host_stderr_txt, current_directory_override=workdir_in_container, stdlib_id=stdlib_id)
+        outputs_library = ToilWDLStdLibTaskOutputs(file_store, host_stdout_txt, host_stderr_txt, current_directory_override=workdir_in_container)
```

Comment:
@stxue1 If we don't pass the directory to ID mapping between the different standard library instantiations, is it possible to get two different standard library instances to upload two sibling files, but with different source directory UUIDs? Maybe by abusing the string-to-File logic?

Reply:
I think string-to-file logic will probably not support sibling files since the standard library instances might be different. The directory to ID mappings would probably have to be passed around in order to support this.